netmanager/src/redux/MapData/operations.js

```diff
@@ -40,6 +45,23 @@ export const loadPM25HeatMapData = () => async (dispatch) => {
     });
 };
 
+export const loadMapSensorsData = () => async (dispatch) => {
```

Comment:
Since the intent of this method is to load all the partners' sensor data, how about we change the method name to something much more descriptive like `loadAllPartnersMapSensorsData`?

Reply:
Done with renaming @Codebmk .